example_site/src/components/AccountInfoForm.js

```diff
@@ -1,16 +1,30 @@
 import { useForm } from "react-hook-form";
+import React, { useState } from "react";
 
 import { useAppContext } from "./AppProvider";
 
 export default function AccountInfoForm({
   account,
-  onSubmit,
+  updateAccount,
```

Comment:
It doesn't look like this prop name was updated in all of the places `<AccountInfoForm />` is rendered. 

But I think that we should revert it back to its original name. Remember that this form is used for updating accounts _and_ creating them, so `updateAccount` is specific to a single use case, whereas `onSubmit` (or `onFormSubmit` or something similar) encompasses whatever this form might be used for.